src/detect-tcp-window.c

```diff
@@ -41,7 +41,7 @@
 /**
  * \brief Regex for parsing our window option
  */
-#define PARSE_REGEX  "^\\s*([!])?\\s*([0-9]{1,9}+)\\s*$"
+#define PARSE_REGEX "^\\s*([!])?\\s*([0-9]{1,9}+)\\s*$"
```

Comment:
nit: this change does not belong to this PR

Reply:
@inashivb this change was made when I ran clang. Last time the PR failed the clang check without it. Should I proceed with creating a new PR without this change?

Reply:
hmm i'm not sure why would clang complain about an untouched part of the code. 🤔 
Are you sure you didn't change anything on this line? maybe a space or something? There are several things in the previous PR that might be objectionable to clang and I'd expect clang to fail for those.

Reply:
I can try making a new PR without this change, I'm not sure if I accidentally changed this line myself.